.github/workflows/integration_tests.yml

```diff
@@ -1056,13 +1056,45 @@ jobs:
         id: deploy_keptn_with_keptn
         run: |
           KEPTN_HELM_CHART_FILENAME=$(ls dist/keptn-installer/keptn*.tgz | head -1)
+          KEPTN_INGRESS_TEMPLATE_FILENAME=test/assets/keptn-on-keptn/ingress.yaml
+          ENDPOINTS_TEMPLATE_BASE_PATH=test/assets/keptn-on-keptn
           HELM_SERVICE_HELM_CHART_FILENAME=$(ls dist/keptn-installer/helm-service*.tgz | head -1)
           JMETER_SERVICE_HELM_CHART_FILENAME=$(ls dist/keptn-installer/jmeter-service*.tgz | head -1)
 
           keptn add-resource --project=${{ env.META_KEPTN_KEPTN_PROJECT }} --service=keptn --all-stages --resource=${KEPTN_HELM_CHART_FILENAME} --resourceUri=helm/keptn.tgz
           keptn add-resource --project=${{ env.META_KEPTN_KEPTN_PROJECT }} --service=helm-service --all-stages --resource=${HELM_SERVICE_HELM_CHART_FILENAME} --resourceUri=helm/helm-service.tgz
           keptn add-resource --project=${{ env.META_KEPTN_KEPTN_PROJECT }} --service=jmeter-service --all-stages --resource=${JMETER_SERVICE_HELM_CHART_FILENAME} --resourceUri=helm/jmeter-service.tgz
 
+          stages=('dev' 'hardening' 'production')
+          services=('keptn' 'helm-service' 'jmeter-service')
+          for SERVICE in "${services[@]}"
+          do
+            for STAGE in "${stages[@]}"
+            do
+              export SERVICE
+              export STAGE
+              envsubst < ${ENDPOINTS_TEMPLATE_BASE_PATH}/endpoints-${SERVICE}.yaml > ${ENDPOINTS_TEMPLATE_BASE_PATH}/endpoints-${SERVICE}-${STAGE}.yaml
+              keptn add-resource \
+                --project=${{ env.META_KEPTN_KEPTN_PROJECT }} \
+                --service=${SERVICE} \
+                --stage=${STAGE} \
+                --resource=${ENDPOINTS_TEMPLATE_BASE_PATH}/endpoints-${SERVICE}-${STAGE}.yaml \
+                --resourceUri=helm/endpoints.yaml
+            done
+          done
```

Comment:
Adding endpoints.yaml for every service and every stage is a lot of work... I will create a follow-up issue for the Keptn core team to improve the feature, based on this example.